### DISCUSSION TITLE
Install the FW on V3 for the first time.

### DISCUSSION THREAD

**Discussion by Nutty-69** [bitfieldlabs/afterglow#19] [Q&A] [answered]
Hi

I have some boards on the way and I was wondering how you install the firmware for the first time. I have read the Wiki so it is just a case  of coping the firware from a PC or is there a different method required for the first install.

Also is it designed to work on any machine straight away or do you need to use the config tool ?

Cheers.

**Comment by bitfieldlabs** [comment, marked as answer]
The rp2040 comes with a uf2 bootloader preinstalled. It will show up as mass storage device when plugged into a USB host. The firmware uf2 file can then just be copied onto this device. No other tools needed.
